src/hotspot/share/opto/loopnode.cpp

```diff
@@ -1726,16 +1726,38 @@ void OuterStripMinedLoopNode::adjust_strip_mined_loop(PhaseIterGVN* igvn) {
     igvn->replace_node(outer_le, iff);
     inner_cl->clear_strip_mined();
   }
+  igvn->C->print_method(PHASE_DEBUG, 2);
```

Comment:
Looks like a leftover from debugging. Otherwise, the fix looks good.

Reply:
Good catch. I thought I removed it but obviously didn't. I updated the change. Thanks for the review.